package.json

```diff
@@ -35,7 +35,8 @@
   },
   "scripts": {
     "bootstrap": "lerna bootstrap",
-    "clean:all": "rm -rf ./node_modules && rm -rf ./packages/*/node_modules",
+    "test-rim": "rimraf ./test-rim",
```

Comment:
Opps must not have cleaned this up! 

I was hitting errors before I changed the order deleting the node_module folders & I don't have rimraf installed globally so I was trying to work it out.